src/main/kotlin/com/triple/clubmileage/api/repository/entity/ImageEntity.kt

```diff
@@ -0,0 +1,36 @@
+package com.triple.clubmileage.api.repository.entity
+
+import org.hibernate.annotations.CreationTimestamp
+import org.hibernate.annotations.GenericGenerator
+import org.hibernate.annotations.UpdateTimestamp
+import java.time.LocalDateTime
+import java.time.LocalDateTime.now
+import java.util.*
+import javax.persistence.*
+
+@Table(name = "IMAGE")
+@Entity
+data class ImageEntity(
+
+    @Id
+    @GenericGenerator(name = "uuid2", strategy = "uuid2")
+    @Column(name = "id", columnDefinition = "UUID")
+    val id: UUID,
+
+    @ManyToOne
+    @JoinColumn(name = "reviewId")
+    var reviewEntity: ReviewEntity,
+
+    @Column(name = "isDeleted")
+    var isDeleted: Boolean,
+
+    @CreationTimestamp
+    @Column(name = "createdAt")
+    val createdAt: LocalDateTime,
+
+    @UpdateTimestamp
+    @Column(name = "updatedAt")
+    val updatedAt: LocalDateTime
+) {
+    constructor(id: UUID, reviewEntity: ReviewEntity) : this(id, reviewEntity, false, now(), now())
+}
```

Comment:
```suggestion
data class ImageEntity(

    @Id
    @GenericGenerator(name = "uuid2", strategy = "uuid2")
    @Column(name = "id", columnDefinition = "UUID")
    val id: UUID,

    @ManyToOne
    @JoinColumn(name = "reviewId")
    var reviewEntity: ReviewEntity,

    @Column(name = "isDeleted")
    var isDeleted: Boolean = false,

    @CreationTimestamp
    @Column(name = "createdAt")
    val createdAt: LocalDateTime = now(),

    @UpdateTimestamp
    @Column(name = "updatedAt")
    val updatedAt: LocalDateTime = now()
)
```
생성자를 따로 만들 필요없이 초기값세팅으로 대체할수있을것같습니다~

Reply:
@hans-triple 기본값을 지정하였지만(isDeleted, createdAt, updatedAt), 모든 속성에 맞춰서 생성자가 만들어졌기때문에
default 값이 있는 field들을 넘겨줘야하더라구요.
그래서 secondary constructor를 만들어서 사용하는곳에서 넘기지않도록 처리했습니다.
코틀린에서 조금더 깔끔하게 처리하는 방법이 있을까요?

Reply:
@hans-triple 아 다시 해보니까 제대로 컴파일되네요 😅